processing/src/main/java/io/druid/query/DefaultQueryMetrics.java

```diff
@@ -66,15 +65,6 @@ public void queryType(QueryType query)
   }
 
   @Override
-  public void interval(QueryType query)
```

Comment:
The idea of QueryMetrics is not breaking compatibility in such a way. Please restore emitting intervals. Only new query types may not emit intervals.